subgraphs/aggregator/src/utils/getOrInitializeCollectionDailyDataByCurrency.ts

```diff
@@ -0,0 +1,20 @@
+import { BigInt } from "@graphprotocol/graph-ts";
+import { ZERO_BD, ONE_DAY_BI } from "../../../../helpers/constants";
+import { CollectionByCurrency, CollectionDailyDataByCurrency } from "../../generated/schema";
+
+export function getOrInitializeCollectionDailyDataByCurrency(
```

Comment:
There is no `getOrInitializeCollection`, isn't it?

Reply:
Yeah, that's because initializing Collection/User will increment some entities' users/collections count and we will end up passing too many arguments into the `getOrInitialize` function